tools/dependency/release_dates.py

```diff
@@ -132,19 +183,27 @@ def verify_and_print_release_dates(repository_locations, github_instance):
 
 
 if __name__ == '__main__':
-    if len(sys.argv) != 2:
+    # github action will have 3 args and CI will have 2 args
+    if len(sys.argv) < 1 or len(sys.argv) > 3:
         print('Usage: %s <path to repository_locations.bzl>' % sys.argv[0])
         sys.exit(1)
+    # parsing location and github_action flag with argparse
+    parser = argparse.ArgumentParser()
+    parser.add_argument('location', type=str)
+    parser.add_argument('--cron', action='store_true')
```

Comment:
doesn't this make --cron default true? would this end up changing how the CI action will handle it?

Reply:
nope, i'm wrong, it defaults false (why is that so confusing?)